README.md

```diff
@@ -62,9 +62,9 @@ After you make changes, be sure to build the new CSS files. Do so by running `gu
 
 ## Contributing
 
-This project is open source and contributions are very welcomed. It is also as beginner friendly as possible, so don't be afraid to jump in if you've never contributed to any Git project before! Feel free to reach out if you are new and need help with the process.
+This project is open source and contributions are very welcome. It is also as beginner friendly as possible, so don't be afraid to jump in if you've never contributed to any Git project before! Feel free to reach out if you are new and need help with the process.
```

Comment:
`welcomed` and `welcome` are both correct in this case, but I prefer `welcomed` in this context.